lib/cn/vblock_reader.h

```diff
@@ -70,23 +66,14 @@ struct vblock_desc {
  */
 merr_t
 vbr_desc_read(
-    struct mpool *           ds,
-    struct mpool_mcache_map *map,
-    uint                     maxidx,
-    uint *                   vgroupsp,
-    u64 *                    argv,
-    struct mblock_props *    props,
-    struct vblock_desc *     vblock_desc);
+    const struct kvs_mblk_desc *mblk,
+    struct vblock_desc *vblk_desc);
 
 merr_t
-vbr_desc_update(
-    struct mpool *           ds,
-    struct mpool_mcache_map *map,
-    uint                     maxidx,
-    uint *                   vgroupsp,
-    u64 *                    argv,
-    struct mblock_props *    props,
-    struct vblock_desc *     vblock_desc);
+vbr_desc_update_vgidx(
+    struct vblock_desc *vblock_desc,
+    uint               *vgroupc,
+    u64                *vgroupv);
```

Comment:
uint64_t please!

Reply:
This must have been copied from source that uses u64.  :)